packages/snfoundry/scripts-ts/deploy-contract.ts

```diff
@@ -3,40 +3,37 @@ import path from "path";
 import { networks } from "./helpers/networks";
 import yargs from "yargs";
 import {
-  BlockIdentifier,
   CallData,
-  hash,
   stark,
   RawArgs,
-  constants,
-  ec,
-  validateAndParseAddress,
   transaction,
-} from "starknet";
-import { Network } from "./types";
-import {
-  LegacyContractClass,
-  CompiledSierra,
   extractContractHashes,
+  DeclareContractPayload
 } from "starknet";
+import { Network } from "./types";
 
 const argv = yargs(process.argv.slice(2)).argv;
 const networkName: string = argv["network"];
 
 let deployments = {};
-
 let deployCalls = [];
 
 const { provider, deployer }: Network = networks[networkName];
 
-const declareIfNot_NotWait = async (payload: any) => {
+const declareIfNot_NotWait = async (payload: DeclareContractPayload, options?: { maxFee?: bigint}) => {
```

Comment:
this is a `maxDeclareFee` rather than a `maxFee` at the moment.

Reply:
we can add another optional `maxFee` [here](https://github.com/ikemHood/scaffold-stark-2/blob/3d3aa29d02512650c459b8c38cfab4181be4bb1a/packages/snfoundry/scripts-ts/deploy-contract.ts#L178-L179) 

```javascript
const executeDeployCalls = async () => {

```

and this maxFee will be set up once inside `deploy.ts` and passed to this function when its called, should be nullable item and add instructions on `deploy.ts`

Reply:
`executeDeployCalls ` usage doesn't make it straight forward to add this params, but i have added it regardless.

another option can be to set the options from deployContract to a variable just like the deployCalls and access it inside the function. Let me know if i should implement this